app/assets/javascripts/i18n.js

```diff
@@ -857,12 +854,43 @@
     }
 
     return scope;
-  }
+  };
+  /**
+   * Merge obj1 with obj2
+   * @param {Object} obj1 Default settings
+   * @param {Object} obj2 User obj2
+   * @returns {Object} Merged values of obj1 and obj2
+   */
+  I18n.extend = function ( obj1, obj2 ) {
+    var extended = {};
+    var prop;
+    for (prop in obj1) {
+      if (Object.prototype.hasOwnProperty.call(obj1, prop)) {
```

Comment:
Just wondering what versions of JS (ES3, ES5, ES6...) supports `Object.prototype` & `hasOwnProperty`